msg/templates/uorb_microcdr/msg.cpp.template

```diff
@@ -151,6 +147,7 @@ void serialize_@(topic_name)(const struct @(uorb_struct) *input, char *output, u
 
     resetStaticMicroCDRForSerialize(microCDRWriter);
 
+    serializeUnsignedLong(input->timestamp, microCDRWriter);
```

Comment:
Should this not be `serializeUnsignedLongLong`?

Reply:
No, is the correct one. Please, see the function [prototype](https://github.com/eProsima/micro-CDR/blob/50776f4ab7895e47af535a6d6e7ec697ddf443f3/src/c/microCdr.c#L1018)

Reply:
But then https://github.com/PX4/Firmware/pull/8888/files#diff-8190543ff07160bad612518e04f0e097R73 should be `uint64_t`

Reply:
No, there is no uint64_t type in IDL

Reply:
Ah, I see, that's IDL. You're right then.